velox/exec/Window.h

```diff
@@ -165,7 +167,7 @@ class Window : public Operator {
 
   // Used to access window partition rows and columns by the window
   // operator and functions. This structure is owned by the WindowBuild.
-  std::unique_ptr<WindowPartition> currentPartition_;
+  std::shared_ptr<WindowPartition> currentPartition_;
```

Comment:
@aditi-pandit It seems we still need use the shared_ptr here. Because we need to hold the WindowPartition in RowStreamingWindowBuild to add remaining Rows when adding input rows [here](https://github.com/facebookincubator/velox/pull/9025/files#diff-99c466a988d2128b820dc950cd0991407ff75f11e52b1e01e2588cf647c52b3eR77).